src/main/resources/edu/cornell/kfs/module/ar/cu-spring-ar.xml

```diff
@@ -72,6 +73,11 @@
           p:code="13"
           p:financialSystemDocumentService-ref="financialSystemDocumentService"
           p:contractsGrantsInvoiceDocumentDao-ref="contractsGrantsInvoiceDocumentDao"/>
+          
+	<bean id="autoApproveDisabledSuspensionCategory" parent="autoApproveDisabledSuspensionCategory-parentBean"/>
```

Comment:
formatting looks a bit off on this line.

Reply:
OK, will fix